dataland-frontend/tests/e2e/specs/prepopulation/Prepopulation.ts

```diff
@@ -1,6 +1,14 @@
-import { retrieveDataIdsList, retrieveCompanyIdsList } from "@e2e/utils/ApiUtils";
-import { doThingsInChunks } from "@e2e/utils/Cypress";
-import { CompanyInformation, EuTaxonomyDataForNonFinancials, EuTaxonomyDataForFinancials } from "@clients/backend";
+import { doThingsInChunks, wrapPromiseToCypressPromise } from "@e2e/utils/Cypress";
```

Comment:
mir ist gerade aufgefallen das auch dieser Test im wesentlichen aus zwei fast identischen Blöcken besteht (analog zu previsit). Auch hier kann man vermutlich einige redundate Blöcke in Funktionen auslagern.